docker/k8s-glue/glue-build-gcp/Dockerfile

```diff
@@ -8,15 +8,18 @@ ENV LANG=en_US.UTF-8
 ENV LANGUAGE=en_US.UTF-8
 ENV PYTHONIOENCODING=UTF-8
 
-COPY ../build-resources/setup.sh /root/setup.sh
+COPY ./setup.sh /root/setup.sh
 RUN /root/setup.sh
 
 COPY ./setup_gcp.sh /root/setup_gcp.sh
-RUN /root/setup_gcp.sh
+RUN chmod +x /root/setup_gcp.sh && /root/setup_gcp.sh
 
-COPY ../build-resources/entrypoint.sh /root/entrypoint.sh
+COPY ./entrypoint.sh /root/entrypoint.sh
 COPY ./provider_entrypoint.sh /root/provider_entrypoint.sh
-COPY ./build-resources/k8s_glue_example.py /root/k8s_glue_example.py
+COPY ./k8s_glue_example.py /root/k8s_glue_example.py
 COPY ./clearml.conf /root/clearml.conf
 
+#TODO: should be adjusted
+RUN python3 -m pip uninstall clearml-agent -y && python3 -m pip install git+https://github.com/FeU-aKlos/clearml-agent.git@feature/support_multi_queeus_with_k8s
```

Comment:
there is still the dockerimage linked, which i created my self, I was not sure how to handle the thing, that the clearml-agent python package should be installed, for usage in the helm chart, without the actual (official) repo having the changes. Any Ideas on that? Should i adjust the setup.sh script and reference my fork there for pip install or the main branch, if i expect, that this PR will be accepted...

Reply:
or in the entrypoint.sh

Reply:
Don't leave your repo here - once we merge the PR we'll build a new image and publish it

Reply:
Okay, great, thanks for clarification